src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/IModelBinder.cs

```diff
@@ -6,22 +6,26 @@
 namespace Microsoft.AspNet.Mvc.ModelBinding
 {
     /// <summary>
-    /// Interface for model binding.
+    /// Defines an interface for model binders.
```

Comment:
Wow, thanks for volunteering to do the sweep of our code base to ensure we use "Defines" everywhere! :trollface:

Seriously though, you're right. "Defines" is nicer, and we should use it consistently, but we currently don't.